src/Composer/ComposerizeDrupalCommand.php

```diff
@@ -255,9 +255,25 @@ protected function mergeTemplateGitignore()
     protected function requireDrupalCore($root_composer_json)
     {
         $version_constraint = DrupalInspector::getVersionConstraint($this->drupalCoreVersion, $this->input->getOption('exact-versions'));
-        $root_composer_json->require->{'drupal/core'} = $version_constraint;
-        $this->getIO()
-            ->write("<info>Added drupal/core $version_constraint to requirements.</info>");
+        if (version_compare($this->drupalCoreVersion, '8.8.0', '<')) {
```

Comment:
Shouldn't we just always force an update to 8.8.0+ ?

Reply:
Sure. that means composerize will support only 8.8.+.